operators/tokenizer/bpetokenizer.hpp

```diff
@@ -1,11 +1,12 @@
 // Licensed under the MIT License.
 // Partial code comes from other Microsoft employee.
 #pragma once
+#include "ocos.h"
 #include "ustring.h"
 
 #include <regex>
+#include <list>
```

Comment:
Perfect - was going to say this is what caused the ci failures in the previous run, should be fine now.